### ISSUE TITLE
 Minipage vertical dimension is not respected

### ISSUE THREAD

**Issue by eccmsilva** [arXiv/html_feedback#2549]
### Description

Figure 4 in https://arxiv.org/html/2410.21228v1 contains a minipage environment, which is being rendered in a zero-height box. The contents of the minipage are spilling over the rest of the page.

### (Required) What is this issue most closely related to? Select one.

Figures

### Internal issue ID

67b3d6e3-edd4-4b78-a8e3-7d657ba29778

### Paper URL

https://arxiv.org/html/2410.21228v1/

### Browser

Chrome/130.0.0.0

### Device Type

Desktop

**Comment by eccmsilva** [comment]
The minipage is showing as a `foreignobject` element in an embedded svg. The height of the element is not being considered when calculating the height of the svg:

```svg
<foreignObject color="#000000" height="63.65" overflow="visible" transform="matrix(1 0 0 -1 0 16.6)" width="557.86">
<span class="ltx_inline-block ltx_minipage ltx_align_bottom" id="S3.F4.pic1.1.1.1.1.1" style="width:403.2pt;">
<span class="ltx_p" id="S3.F4.pic1.1.1.1.1.1.1"><span class="ltx_text ltx_font_bold" id="S3.F4.pic1.1.1.1.1.1.1.1">Algorithm: Finding Intruder Dimensions.</span></span>
<span class="ltx_listing ltx_minipage ltx_align_center ltx_align_middle ltx_listing" id="S3.F4.pic1.1.1.1.1.1.2" style="width:429.3pt;">
...
```

**Comment by dginev** [comment]
@eccmsilva great diagnostic, exactly right. 

We need better latexml support for sizing in tcolorbox.sty. 
This happens to be the most requested latexml-related arXiv issue today with 200 open reports in this tracker.

**Comment by eccmsilva** [comment]
Duplicate of #2155 and many others.

**Comment by dginev** [comment]
> Shouldn't you start closing the duplicates?

Bookkeeping them is more work than is useful to spend. Also, the UI component on the articles doesn't have GH visibility, so reporters don't know of previous reports and we'll keep receiving duplicates.

I am keeping a separate aggregate report for now - my current intention is to "mass close" relevant issues on resolution.

**Comment by dginev** [comment]
> What is the corresponding issue in the LaTeXML repository?

I don't think there is one targeted at this package as of yet, but there is a dedicated label for pgf/tikz reports that are more atomic:

https://github.com/brucemiller/LaTeXML/issues?q=is%3Aissue+is%3Aopen+tikz+label%3Apgf%2Ftikz

I don't think we'll resolve this before latexml 0.8.9 ships btw, there are a couple of queued up large refactors that need to land first.
